engine/Gemfile

```diff
@@ -6,7 +6,7 @@ gemspec
 
 group :development, :test do
   gem "capybara"
-  gem "citizens-advice-style", github: "citizensadvice/citizens-advice-style-ruby", tag: "v2.1.0"
+  gem "citizens-advice-style", github: "citizensadvice/citizens-advice-style-ruby", tag: "v7.0.0"
```

Comment:
This big jump was fine because we're already using `rubocop-rspec` and the engine ruby code is pretty simple.